docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+```
+## Deploying to Polygon Mainnet
+
+The method to deploy onto mainnet is the exact same as with Mumbai, with a few key items to note. Your `url`, `lspCreatorAddress` and `financialProductLibrary` will need to be amended for use on Polygon mainnet. Since the library addresses are different between the networks. 
+
+All the required Polygon Mainnet addresses can be seen on our [Github repo](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json) however to give you a quick reference here are a few key contracts to use. 
+
+- `Polygon LongShortPairCreator:` 0x3e665D15425fAee14eEF53B9caaa0762b243911a
+- `BinaryOptionLongShortPairFinancialProductLibrary:` 0xda768D869f1e89ea005cde7e1dBf630ff9307F33
+- `CoveredCallLongShortPairFinancialProductLibrary:` 0x3F62D7F4Be7671cc93BCDFE7A3Dd900FEC4b5025
+- `LinearLongShortPairFinancialProductLibrary:` 0xcFF28e9E83cEc1BCa8D8619dC7eA60244b433502
+- `RangeBondLongShortPairFinancialProductLibrary:` 0x7A9Bbd278b40f90F1269cB3a9D94a63333febdD4
+
+Please note that the script will deploy to the network that your node URL is for. If using Infura for a Polygon deployment, `YOUR_NODE_URL` will follow this format:
+
+
+```bash
+wss://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+If using Infura for a mainnet fork, `YOUR_NODE_URL` will follow this format:
+
+```bash
+https://polygon-mainnet.infura.io/v3/{projectId}
+```
+
+You can now run the deployment script. From within the `launch-lsp` directory, run:
+```bash
+node index.js --gasprice 20 --url YOUR_NODE_URL --mnemonic "your mnemonic (12 word seed phrase)" --lspCreatorAddress 0x3e665D15425fAee14eEF53B9caaa0762b243911a --expirationTimestamp 1643678287 --collateralPerPair 1000000000000000000 --priceIdentifier ETHUSD --collateralToken 0xd0a1e359811322d97991e03f863a0c30c2cf029c --syntheticName "ETH 9000 USD Call [December 2021]" --syntheticSymbol ETHc9000-1221 --financialProductLibrary 0x3F62D7F4Be7671cc93BCDFE7A3Dd900FEC4b5025
+```
+Once deployed, the script will list the address of your newly deployed EMP. A successful output will look like this:
+
+```bash
+Simulating Deployment...
+Simulation successful. Expected Address: 0x44978157afE92c926619EBB54599bbc483eBe871
+``` 
+## Linking your Financial Product Library
+
+Once your contract is deployed, you can head over the the specific Financial Product Library your contract wants to use on [Polygonscan](https://polygonscan.com/). You need to link your newly deployed contract to the Financial Product Library. 
+
+In the case of this launch demo, we used the `CoveredCallLongShortPairFinancialProductLibrary:` 0x3F62D7F4Be7671cc93BCDFE7A3Dd900FEC4b5025 and will link the new contract with a given strike price. 
+
+![](/img/PolygonFPL.png)
```

Comment:
Can you explain what these parameters are? Also in the image, you show a strikePrice of 100 which would be a strike price of `0.0000000000000001` for ETHUSD, which may be a little confusing.

Reply:
Cool can change this image